lib/chef/provider/group/suse.rb

```diff
@@ -51,7 +52,12 @@ def set_members(members)
         end
 
         def add_member(member)
-          shell_out!("groupmod -A #{member} #{@new_resource.group_name}")
+          begin
+            Etc.getpwnam(member)
+            shell_out!("groupmod -A #{member} #{@new_resource.group_name}")
+          rescue ArgumentError => e
+            raise Chef::Exceptions::Group, "Could not add #{member} to #{@new_resource.group_name}: user does not exist"
```

Comment:
you should probably add at least a `def to_add { members - @current_resource.members }` which you can call from `define_resource_requirements` and then `to_add.all? { |member| Etc.getpwnam(member) }` with some kinda rescue that fails it more or less.